pkg/controller/storageprofile-controller.go

```diff
@@ -71,18 +75,9 @@ func (r *StorageProfileReconciler) Reconcile(_ context.Context, req reconcile.Re
 		return reconcile.Result{}, err
 	}
 
-	// Alert for incomplete profiles
-	numIncomplete := 0
-	storageProfileList := &cdiv1.StorageProfileList{}
-	if err := r.client.List(context.TODO(), storageProfileList); err != nil {
+	if err := r.checkIncompleteProfiles(); err != nil {
```

Comment:
maybe you can shorten to one line:
return reconcile.Result{}, r.checkIncompleteProfiles()

Reply:
Done